content/SCALECLIReference/Auth/CLIApiKey.md

```diff
@@ -10,6 +10,193 @@ tags:
+
+### Create Command
+
+The `create` command lets you create simple or complex API keys.
+
+{{< expand "Creating a Simple API Key" "v" >}}
+
+#### Description
+The `create` command requires the `name` property.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns the API key when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key create name=<i>name</i></code>
+
+Where *name* is the name you want to assign to the key.
+
+{{< expand "Command Example" "v" >}}
+```
+auth api_key create name=apikey3
+API Key: 3-xTqwhyf3SrUgUlotMQEEGuUr6oRvqg89SBDfXob6xtWSgLbRiDBr6SVRWxswSXx3
+```
+{{< /expand >}}
+{{< /expand >}}
+
+{{< expand "Creating a Complex API Key" "v" >}}
+
+#### Description
+The `create` command requires the `name` property. You can also specify an HTTP method and a WebSocket API method.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns the API key when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key create name=<i>name</i> allowlist=[{"method":"<i>METHOD</i>","resource":"<i>api.resource</i>"}]</code>
+
+Where 
+* *name* is the name you want to assign to the key.
+* *METHOD* is the HTTP method you want the key to use. Options are GET, POST, PUT, DELETE, CALL, and SUBSCRIBE.
+* *api.resource* is the WebSocket API resource you want to use. Append "/api/docs/" to the end of your TrueNAS web UI address to see our full list of WebSocket API resources.
+
+{{< expand "Command Example" "v" >}}
+```
+auth api_key create name=apikey3 allowlist=[{"method":"SUBSCRIBE","resource":"certificate.query"}]
+API Key: 3-xTqwhyf3SrUgUlotMQEEGuUr6oRvqg89SBDfXob6xtWSgLbRiDBr6SVRWxswSXx3
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Delete Command
+
+The `delete` command deletes an API key.
+
+{{< expand "Deleting an API Key" "v" >}}
+
+#### Description
+The `delete` command requires the `id` property.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns nothing when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key delete id=<i>number</i></code>
+
+Where *number* is the list number of the API key you want to delete. For example, the first API key created on the system would be "1".
+{{< expand "Command Example" "v" >}}
+```
+auth api_key delete id=1
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Get_Instance Command
+
+The `get_instance` command returns a table of properties for the specified API key.
+
+{{< expand "Viewing API Key Instances" "v" >}}
+
+#### Description
+The `get_instance` command requires the `id` property.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table of properties for the specified API key when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key get_instance id=<i>number</i></code>
+
+Where *number* is the list number of the API key you want to delete. For example, the first API key created on the system would be "1".
```

Comment:
the second sentence is better as part of the Description in a line that explains the required id property.